File: ETAPA_2 GIT Y JAVA/Dia 4 a 6/Estructuras de control/Java1 dia 4 a 6/src/EjerciciosGuia3/Ejercicio4.java
package EjerciciosGuia3;


import java.util.Scanner;

/*
Escriba un programa que pida una frase o palabra y valide si la primera letra
de esa frase es una ‘A’. Si la primera letra es una ‘A’, se deberá de imprimir 
un mensaje por pantalla que diga “CORRECTO”, en caso contrario, se deberá
imprimir “INCORRECTO”. Nota: investigar la función Substring y equals() de Java.

 */

/**
 *
 * @author Sol
 */
public class Ejercicio4 {

    /**
     * @param args the command line arguments
     */
    public static void main(String[] args) {
        Scanner leer = new Scanner (System.in);
        System.out.println("Ingrese una frase para determinar si la primera letra es A");
        String frase = leer.nextLine();
        String letra = frase.substring(0,1);
        letra=letra.toUpperCase();
        if(letra.equals("A")) {
            System.out.println("CORRECTO");
        }else{
            System.out.println("INCORRECTO");
        }
       
    }
    
}

File: ETAPA_4 FRONT/2 - JavaScript/Ejercicios/ejercicio1/ejercicio4.js
// Realiza un programa que sólo permita introducir los caracteres ‘S’ y ‘N’. Si el usuario
// ingresa alguno de esos dos caracteres se deberá de imprimir un mensaje por pantalla
// que diga “CORRECTO”, en caso contrario, se deberá imprimir “INCORRECTO”.

var letra = prompt("Ingrese S o N");

if(letra.toLowerCase()==="s" || letra.toLowerCase()==="n") {
    alert("CORRECTO");
} else {
    alert("INCORRECTO");
};